validator/client/validator.go

```diff
@@ -937,6 +937,11 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 
 // UpdateFeeRecipient calls the prepareBeaconProposer RPC to set the fee recipient.
 func (v *validator) UpdateFeeRecipient(ctx context.Context, km keymanager.IKeymanager) error {
+	// only used after Bellatrix
+	if slots.ToEpoch(slots.CurrentSlot(v.genesisTime)) < params.BeaconConfig().BellatrixForkEpoch {
```

Comment:
We don't really need it until the bellatrix fork epoch is set

```suggestion
	e := params.BeaconConfig().BellatrixForkEpoch 
	if e != math.MaxUint64  && slots.ToEpoch(slots.CurrentSlot(v.genesisTime)) < e{
```